src/mp-instance.js

```diff
@@ -229,7 +247,12 @@ export default function mParticleInstance(instanceName) {
             self.setDeviceId(guid);
         }, self);
         if (queued) return;
-        this._Persistence.setDeviceId(guid);
+
+        // debugger;
+        // TODO: Replace this with Store.setDeviceId
```

Comment:
This could be a blanket TODO. Anywhere where something shouldbe saved to persistence but persistence is off, we should instead save to store.

Reply:
I think that could be done as a second ticket though. The first ticket is to get the SDK working and not crash when persistence is turned off.  The second ticket could then be save to/read from store instead.